setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = oldest-supported-numpy
-description = Meta-package that provides the oldest Numpy for given Python version and platform
+description = Meta-package that provides the oldest Numpy for given Python version and platform. Where possible, this relfects the wheels released via PyPI.
```

Comment:
This still has a typo. More importantly, the added sentence isn't all that clear. How about changing this whole description to:
_"Meta-package that provides the oldest NumPy that supports a given Python version and platform. If wheels for the platform became available on PyPI only for a more recent NumPy version, then that NumPy version is specified."_

Reply:
adopted